ui/app/components/allocation-row.js

```diff
@@ -106,7 +106,7 @@ async function qualifyAllocation() {
     // Make sure that the job record in the store for this allocation
     // is complete and not a partial from the list endpoint
     const job = allocation.get('job.content');
-    if (job) await job.reload();
+    if (job.isPartial) await job.reload();
```

Comment:
More [context](https://github.com/hashicorp/nomad/commit/8f811d50463774233e95c01cd019b542c15c0552#r85482632) on this solution.